src/routes/api/cryptor-migrations/migration-installation.ts

```diff
@@ -29,20 +33,23 @@ export const CryptorMigrationInstallationPost = async (req: Request, res: Respon
 
 	//--------- loop and save each one of them --------------
 	let count = 0;
+	let lastInst: Installation | null = null;
```

Comment:
I dont' see the point of this, if all you want is the ID.  Why not just store the `lastInstallationId` instead?

Reply:
Agree we could store either `lastInst` or `lastId`. But feels like really minor here, so not changing it.